lib/LWP/Protocol/https.pm

```diff
@@ -129,6 +129,7 @@ sub _get_sock_info
     my $self = shift;
     $self->SUPER::_get_sock_info(@_);
     my($res, $sock) = @_;
+    $sock->can('get_sslversion') and $res->header("Client-SSL-Version" => $sock->get_sslversion);
```

Comment:
It looks like get_sslversion() may return undef. So how about
```diff
diff --git lib/LWP/Protocol/https.pm lib/LWP/Protocol/https.pm
index 42451c6..f8fe07e 100644
--- lib/LWP/Protocol/https.pm
+++ lib/LWP/Protocol/https.pm
@@ -129,7 +129,7 @@ sub _get_sock_info
     my $self = shift;
     $self->SUPER::_get_sock_info(@_);
     my($res, $sock) = @_;
-    $sock->can('get_sslversion') and $res->header("Client-SSL-Version" => $sock->get_sslversion);
+    $sock->can('get_sslversion') and $res->header("Client-SSL-Version" => $sock->get_sslversion || "Unknown");
     $res->header("Client-SSL-Cipher" => $sock->get_cipher);
     my $cert = $sock->get_peer_certificate;
     if ($cert) {
```

Reply:
@skaji @oalders Good point. Perhaps it would be better not to add the header at all when it's unknown, as with Net::SSL? That seems better than the arbitrary "Unknown" to me. E.g. something like:

```
$sock->can('get_sslversion')
    and my $sslversion = $sock->get_sslversion
    and $res->header("Client-SSL-Version" => $sslversion);
```

Reply:
Sure, that sounds good to me.  @skaji?

Reply:
Sounds reasonable.

Reply:
@skaji @oalders Ok, new version pushed. My untested proposal above had a `my` scope problem, so this is slightly different.